docs/reference/setup/configuration.asciidoc

```diff
@@ -98,7 +98,8 @@ You can configure and update dynamic settings on a running cluster using the
 <<cluster-update-settings,cluster update settings API>>.
```

Comment:
> You can configure and update dynamic settings...

It's covered in the API, but is it also worth highlighting here that dynamic settings can be either persistent (apply across restarts) or transient (don’t survive a full cluster restart)?  That's something you don't get with static settings.